crates/host/src/wasmbus/mod.rs

```diff
@@ -2275,7 +2349,14 @@ impl Host {
             ..
         }: async_nats::Message,
     ) {
-        let mut parts = subject.split('.').skip(3); // skip `wasmbus.ctl.{prefix}`
+        // Skip the topic prefix and then the lattice prefix
+        // e.g. `wasmbus.ctl.{prefix}`
+        let mut parts = subject
+            .trim()
+            .trim_start_matches(&self.ctl_topic_prefix)
+            .trim_start_matches('.')
```

Comment:
Nit: Could you just go straight to the `split` and then skip 2 to skip the empty string and then prefix?

Reply:
Do you mean something like `subject.split(&self.ctl_topic_prefix).skip(1).split('.')`? I did it this way because someone could pick a prefix with an arbitrary number of parts